packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -200,38 +203,162 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
+  /// example, wires this up in [WidgetsBinding.wrapWithDefaultView], which is
+  /// called indirectly from [runApp].
+  ///
+  /// Apps, that don't use the [WidgetsBinding] or don't call [runApp] (or
+  /// [WidgetsBinding.wrapWithDefaultView]) must manually add this pipeline owner
+  /// to the pipeline owner tree rooted at [rootPipelineOwner] and assign a
+  /// [RenderView] to it if the they want to use this deprecated property.
+  ///
+  /// Instead of accessing this deprecated property, consider interacting with
+  /// the root of the [PipelineOwner] tree (exposed in [rootPipelineOwner]) or
+  /// instead of accessing the [SemanticsOwner] of any [PipelineOwner] consider
+  /// interacting with the [SemanticsBinding] (exposed via
+  /// [SemanticsBinding.instance]) directly.
+  @Deprecated(
+    'Interact with the pipelineOwner tree rooted at RendererBinding.rootPipelineOwner instead. '
+    'Or instead of accessing the SemanticsOwner of any PipelineOwner interact with the SemanticsBinding directly. '
+    'This feature was deprecated after v3.10.0-12.0.pre.'
+  )
+  late final PipelineOwner pipelineOwner = PipelineOwner(
+    onSemanticsOwnerCreated: () {
+      (pipelineOwner.rootNode as RenderView?)?.scheduleInitialSemantics();
+    },
+    onSemanticsUpdate: (ui.SemanticsUpdate update) {
+      (pipelineOwner.rootNode as RenderView?)?.updateSemantics(update);
+    },
+    onSemanticsOwnerDisposed: () {
+      (pipelineOwner.rootNode as RenderView?)?.clearSemantics();
+    }
+  );
+
+  /// Deprecated. Will be removed in a future version of Flutter.
+  ///
+  /// This is typically the root of the render tree bootstrapped by [runApp].
+  ///
+  /// However, by default this render view is not associated with any
+  /// [PipelineOwner] and therefore isn't considered during frame production.
+  /// It is also not registered with this binding via [addRenderView].
+  /// Wiring this up is left as a responsibility for a higher level. The
+  /// [WidgetsBinding], for example, sets this up in
+  /// [WidgetsBinding.wrapWithDefaultView], which is called indirectly from
+  /// [runApp].
+  ///
+  /// Apps, that don't use the [WidgetsBinding] or don't call [runApp] (or
+  /// [WidgetsBinding.wrapWithDefaultView]) must manually assign a
+  /// [PipelineOwner] to this [RenderView], make sure the pipeline owner is part
+  /// of the pipeline owner tree rooted at [rootPipelineOwner], and call
+  /// [addRenderView] if they want to use this deprecated property.
+  ///
+  /// Instead of interacting with this deprecated property, consider using
+  /// [renderViews] instead, which contains all [RenderView]s managed by the
+  /// binding.
+  @Deprecated(
+    'Consider using RendererBinding.renderViews instead as the binding may manage multiple RenderViews. '
+    'This feature was deprecated after v3.10.0-12.0.pre.'
+  )
+  late final RenderView renderView = _ReusableRenderView(
+    view: platformDispatcher.implicitView!,
```

Comment:
What happens if I use an embedder that doesn't provide an implicit view? If that would result in an error, should we call this out in the comment?

Reply:
This line and `wrapWithDefaultView` will have to be updated to use a view created with the windowing API, once we have that. I do not expect this to crash.